Axioms and dimer class

giacomodecolle opened the pull request:
Added the new definitions for secondary and tertiary conformation, previously discussed in the call with Lauren and Morgan. Axioms for processes are added. A class for the EGFR dimer as a material entity is added. Some issues with one class are communicated via email.

giacomodecolle committed:
Axioms and dimer class

Added the new definitions for secondary and tertiary conformation, previously discussed in the call with Lauren and Morgan. Axioms for processes are added. A class for the EGFR dimer as a material entity is added. Some issues with one class are communicated via email.

addiehl commented on PRC.owl:
```diff
@@ -301,7 +301,7 @@
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/PRC.owl/PRC_0000002">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/PRC.owl/PRC_0000001"/>
-        <obo:IAO_0000115>A protein conformation that is realized in the formation of hydrogen bonds between the partially negative oxygen atom and partially positive nitrogen atom in the polypeptide backbone of amino acids and the specific 3-dimensional structure that results.</obo:IAO_0000115>
+        <obo:IAO_0000115>A protein conformation that is realized in the formation of hydrogen bonds between the partially negative oxygen atoms and partially positive nitrogen atoms in one or more polypeptide backbones of amino acids and the specific 3-dimensional structure that results. Beta sheets in particular may involve the same or separate amino acid chains, whether most other secondary str ...
```
Change the "one or more polypeptide backbones of amino acids" to "one or more amino acid chains" as this is more consistent with usage in the Protein Ontology and PRC.

Change the word "whether" to "whereas".

addiehl replied:
Also add your ORCID and name as dc:creator.

addiehl commented on PRC.owl:
```diff
@@ -1520,7 +1520,7 @@
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/PRC.owl/PRC_0000088">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/PRC.owl/PRC_0000001"/>
-        <obo:IAO_0000115>A protein conformation that is realized in the arranging of secondary structure elements and that results in a 3-dimensional shape.</obo:IAO_0000115>
+        <obo:IAO_0000115>A protein conformation that is realized in the arranging of secondary structure elements of a single amino acid chain and that results in a 3-dimensional shape.</obo:IAO_0000115>
```
Add your ORCID and name as a dc:creator.

giacomodecolle committed:
Update PRC.owl

addiehl commented on PRC.owl:
```diff
@@ -301,11 +301,13 @@
     <owl:Class rdf:about="http://purl.obolibrary.org/obo/PRC.owl/PRC_0000002">
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/PRC.owl/PRC_0000001"/>
-        <obo:IAO_0000115>A protein conformation that is realized in the formation of hydrogen bonds between the partially negative oxygen atom and partially positive nitrogen atom in the polypeptide backbone of amino acids and the specific 3-dimensional structure that results.</obo:IAO_0000115>
+        <obo:IAO_0000115>A protein conformation that is realized in the formation of hydrogen bonds between the partially negative oxygen atoms and partially positive nitrogen atoms in one or more amino acid chains and the specific 3-dimensional structure that results. Beta sheets in particular may involve the same or separate amino acid chains, whereas most other secondary structures involve onl ...
```
A protein conformation that is realized in the formation of hydrogen bonds between the partially negative oxygen atoms and partially positive nitrogen atoms in the backbones of one or more amino acid chains and the specific 3-dimensional structure that results.

giacomodecolle committed:
Update PRC.owl

Added two beta sheets classes from Morgan and changed secondary structure definition.
